doc/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = python $(shell which sphinx-build)
```

Comment:
I keep needing to add this to build the docs in a venv. Is it just me?

Reply:
I get `sphinx-build` on the `$PATH` in a venv. But I've had to do essentially this in order to debug sphinx. Fine by me.